src/helm/proxy/clients/client.py

```diff
@@ -109,6 +115,58 @@ def truncate_sequence(sequence: Sequence, request: Request, print_warning: bool
     return sequence
 
 
+def truncate_and_tokenize_response_text(
```

Comment:
Is anyone else going to use this except mistral-medium?

Reply:
Yes, we should migrate Anthropic Claude, Google Gemini, Amazon Titan etc. to use this method too.

Reply:
Actually, maybe just Claude because we don't have a tokenizer for Gemini or Titan (sigh)